config.js

```diff
@@ -7,8 +7,8 @@ export default {
     "rpc_url": process.env.RPC_URL,
     "private_key": process.env.PRIVATE_KEY,
     "account_type": process.env.ACCOUNT_TYPE || 'simple',
-    "contract_address": "0x66519FCAee1Ed65bc9e0aCc25cCD900668D3eD49",
-    "function_name": "mintTo",
-    "entry_point": "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789",
-    "entry_point_version": "0.6"
+    "contract_address": "0x7d662A03CC7f493D447EB8b499cF4533f5B640E2",
+    "function_name": "click",
+    "entry_point": "0x0000000071727De22E5E9d8BAf0edAc6f37da032",
+    "entry_point_version": "0.7"
```

Comment:
`contract_address` & `function_name` - replace with your dapp's smart contract functions

`entry_point` & `entry_point_version` - replace with whatever entrypoint you want to use (v06 vs v07)